java/main/com/google/privacy/differentialprivacy/DpPreconditions.java

```diff
@@ -143,4 +143,11 @@ static void checkMergeMechanismTypesAreEqual(MechanismType type1, MechanismType
     checkArgument(Objects.equals(type1, type2),
         "Failed to merge: unequal mechanism types. type1 = %s, type2 = %s", type1, type2);
   }
+
+  static void checkConfidenceLevel(double confidenceLevel){
+    checkArgument(
+            confidenceLevel < 0 || confidenceLevel > 1 || Double.isNaN(confidenceLevel),
```

Comment:
probably check for infinity as well?

Reply:
It is covered in "confidenceLevel > 1". Java evaluates this to true when confidence level is infinite.

Reply:
I see, yes, you are right.